src/main/kotlin/org/rust/ide/inspections/RsDropRefInspection.kt

```diff
@@ -30,7 +33,9 @@ class RsDropRefInspection : RsLocalInspectionTool() {
         val pathExpr = expr.expr as? RsPathExpr ?: return
 
         val resEl = pathExpr.path.reference.resolve()
-        if (resEl !is RsFunction || resEl.crateRelativePath != "::mem::drop") return
+        if (resEl !is RsFunction ||
```

Comment:
The power of [KnownItems](https://github.com/intellij-rust/intellij-rust/blob/3f96f1509a46d4a61bc933df4e070c1c7fdb8a4a/src/main/kotlin/org/rust/lang/core/resolve/KnownItems.kt#L42) can be used here, so this condition simplifies to something like `resEl == pathExpr.knownItems.drop`

Reply:
I'm not sure how can I use it. It doesn't contain a `drop` property and most of its properties are traits while I am checking if the element is a free drop function. Am I missing something here?

Reply:
Hmm, I thought that it's easy to add a `drop `property to KnownItems, but now I see that not =/
It's designed to resolve only resolve `type` paths, so function will not be resolved... So ok, lets leave it as is for now